trap_posix.go

```diff
@@ -11,15 +11,18 @@ import (
 
 func (d *Daemon) trapSignals() {
 	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, syscall.SIGUSR1)
+	signal.Notify(ch, syscall.SIGUSR1, syscall.SIGINT, syscall.SIGTERM, syscall.SIGABRT)
 	for {
 		select {
 		case s := <-ch:
 			switch s {
 			case syscall.SIGUSR1:
 				d.handleSIGUSR1()
+			case syscall.SIGINT, syscall.SIGTERM, syscall.SIGABRT:
+				d.Close()
+				os.Exit(int(s.(syscall.Signal)))
```

Comment:
```suggestion
				os.Exit(0x80 + int(s.(syscall.Signal)))
```

It's customary for processes to exit with `128 + kill signal number` when interrupted by a kill signal. See https://www.tldp.org/LDP/abs/html/exitcodes.html.

Reply:
Done